frontend-styleguide/README.md

```diff
@@ -488,3 +489,69 @@ it("Correctly formats the name", () => {
 ```
 
 [Back to top ⬆️](#pushpin-summary)
+
+
+## Translate
+
+### 👉 Use I18n to translate
+
+Our plataform has support for internationalized (i18n). First step is configurated [nextJs](https://nextjs.org/docs/advanced-features/i18n-routing) to support locales on the routes.
```

Comment:
```suggestion
Our plataform has support for internationalization (i18n). First step is configurate [nextJs](https://nextjs.org/docs/advanced-features/i18n-routing) to support locales on the routes.
```

Reply:
Should be `configure`, not `configurate`

Reply:
Fix into `678a506`